R/object-import.R

```diff
@@ -34,3 +39,19 @@ format.rd_section_reexport <- function(x, ...) {
     "\n}}\n"
   )
 }
+
+find_topic_in_package_reexp <- function(pkg, topic) {
+  path <- tryCatch(
+    find_topic_in_package(pkg, topic),
+    error = function(err) {
+      roxy_warning("Unavailable package in re-export: ", pkg, "::", topic)
+      topic
+    }
+  )
+  if (is.na(path)) {
+    roxy_warning("Unavailable topic in re-export: ", pkg, "::", topic)
+    topic
+  } else {
+    path
```

Comment:
i.e. should this be `paste(pkg, ":", topic)`?

Reply:
Maybe not, because then some callers need to take it apart again?